### ISSUE TITLE
Feature request: TLS Reports

### ISSUE THREAD

**Issue by ReVoLt112** [liuch/dmarc-srg#50]
Thought about implementing TLS Reports? Same principle than DMARC Reports just another parser

**Comment by liuch** [comment]
Yes I thought of that. Unfortunately, it's not just another parser. First I want to implement what I have planned for DMARC. This has already been discussed here: #11
